RedHat / CentOS (7 and 8) support

sergei-maertens opened the pull request:
- [x] Up to date PostgreSQL server installation
- [x] Up to date docker engine installation
- [x] Check nginx version/installation
- [x] Test on CentOS 8

sergei-maertens committed:
:sparkles: Support a maintained PostgreSQL version on RedHat 7

And of course this also means CentOS 7.

sergei-maertens committed:
:pencil: Document newer Docker version

sergei-maertens committed:
:sparkles: Handle differences between RHEL/CentOS 7 & 8

RHEL/CentOS 8 comes with dnf as package manager, and yum no longer
really exists (unless explicitly installed).

sergei-maertens committed:
:pencil: Document Red Hat role

sergei-maertens committed:
:pencil: Document OS Setup role

sergei-maertens committed:
:pencil: Provide high-level documentation on Open Zaak collection

sergei-maertens committed:
:bookmark: Bump version to 0.10.0

joeribekker commented on roles/redhat_setup/README.md:
```diff
@@ -0,0 +1,40 @@
+
+None
+
+Exmaple Playbook
```
```suggestion
Example Playbook
```

sergei-maertens committed:
:ok_hand: Fix typo

Co-authored-by: Joeri Bekker <joeri@maykinmedia.nl>

sergei-maertens committed:
:sparkles: Extra tasks for CentOS 8 installation

The PostgreSQL setup required more repositories to be enabled, which
wasn't the case on CentOS 7.

Additionally, the setup for CentOS 7 vs. 8 is now split into separate
Ansible files grouping the tasks per version, dynamically included
based on the discovered facts.

sergei-maertens committed:
:pencil: Update RHEL/CentOS docs

sergei-maertens commented:
I've made a couple more changes required for CentOS 8 - it's now tested and functional.

sergei-maertens committed:
:pencil: Update docs for CentOS 8

sergei-maertens committed:
:pencil: Added contributors
